src/Autocomplete/src/EntityAutocompleterInterface.php

```diff
@@ -46,4 +46,9 @@ public function getValue(object $entity): mixed;
      * Note: if SecurityBundle is not installed, this will not be called.
      */
     public function isGranted(Security $security): bool;
+
+    /**
+     * Return group_by option.
+     */
+    public function getGroupBy(): mixed;
```

Comment:
Instead of physically adding this method, which would break BC, add some documentation to the top for it: https://github.com/SymfonyCasts/reset-password-bundle/blob/6601f15fce7a4934bc9570f76feaf1b93536b1a7/src/ResetPasswordHelperInterface.php#L19 - that will allow people to be aware of it, and then we will add the actual method whenever we release a new major version.

Reply:
Yes i think, It can be string, callable or null

Reply:
I'm struggling to see how to normalize this, how do you see the implementation of this normalization